exporter/node.go

```diff
@@ -207,7 +207,7 @@ func (exp *exporter) handleValidatorAddedEvent(event eth1.ValidatorAddedEvent) e
 	logger := exp.logger.With(zap.String("pubKey", pubKeyHex))
 	logger.Info("validator added event")
 	// save the share to be able to reuse IBFT functionality
-	validatorShare, err := validator.ShareFromValidatorAddedEvent(event)
+	validatorShare, err := validator.ShareFromValidatorAddedEvent(event, "")
```

Comment:
how comes that we dont pass operator key here?

Reply:
this is only from exporter where there is no operator key